### ISSUE TITLE
Scan photos in group folders

### ISSUE THREAD

**Issue by tclavier** [matiasdelellis/facerecognition#502]
### Actual behaviour

In my family, we have a group folder for all commons elements like family pictures :-)

### Steps to reproduce

1. Install the Group folders application : https://github.com/nextcloud/groupfolders
1. Configure a Group folder
1. Upload one photo to this group folder

This photo is never scan.

### Expected behaviour

Photo is scan and i can identify faces.

**Comment by matiasdelellis** [comment]
Hi @tclavier 
See:
* https://github.com/matiasdelellis/facerecognition/wiki/Settings#hidden-settings
* https://github.com/matiasdelellis/facerecognition/wiki/Settings#notes

**Comment by tclavier** [comment]
thanks a lot, i thinks this hidden settings could be in settings page :-)

**Comment by isaacolsen94** [comment]
I agree, it would be nice to have a few more of those "hidden" options as well as there warnings added to the gui.
